core/CliMulti.php

```diff
@@ -142,7 +142,7 @@ private function start($piwikUrls)
             if ($shouldStart === Request::ABORT) {
                 // output is needed to ensure same order of url to response
                 $output = new Output($cmdId);
-                $output->write('Skipped');
+                $output->write(serialize(array('skipped' => '1', 'nb_visits' => 0)));
```

Comment:
the output doesn't really matter as long as it is a non-empty serialized array as we check with `empty($metric['nb_visits'])` etc.

Reply:
In theory the response would need to depend on the caller but maybe in this case it isn't too important? checked it is used as well in `ScheduledTaskRunner` but it doesn't use the "abort" feature and the response format doesn't matter there

Reply:
It's not really an error I think. It is just not needed to execute it anymore basically (eg because the same job is already running). Setting `null` could be difficult maybe as it would be hard to differentiate between the aborted case and eg no response received etc. 

So far we call CLI Multi only in Task Runner and Cron Jobs and there it is all fine. The `abort` feature is not really any documented feature for plugins so shouldn't be a problem. Could set it to `$output->write(serialize(array('aborted' => '1')))` though and explain the output in the description of the constant... maybe that makes it more clear. A caller has to specifically implement some support to `ABORT` a command so I don't think it's a problem.

Reply:
Just changed the output... the `ABORT` feature in there is currently actually specifically in there for the archiver anyway.